components/mpas-ocean/src/mode_forward/mpas_ocn_time_integration_si.F

```diff
@@ -55,6 +55,18 @@ module ocn_time_integration_si
    use ocn_surface_land_ice_fluxes
    use ocn_transport_tests
 
+#ifdef MPAS_OPENACC
+#ifdef USE_MAGMA
+   use magma
```

Comment:
I do see some MAGMA sections (example: line 4295) use OpenACC directives. Do you need `use openacc` here or am I missing something?

Reply:
Oh, it's because this is already conditional on MPAS_OPENACC #ifdef. 

In that case, do you need `use openacc` under USE_CUBLAS? I guess I was looking for symmetrical sections for both libs.

Reply:
@sarats , Thanks for catching it! It doesn't need to be there, so I removed it!